alloc/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "soroban-alloc-contract"
+version = "0.0.0"
+authors = ["Stellar Development Foundation <info@stellar.org>"]
+license = "Apache-2.0"
+edition = "2021"
+publish = false
+
+[lib]
+crate-type = ["cdylib"]
+doctest = false
+
+[dependencies]
+soroban-sdk = { workspace = true }
```

Comment:
```suggestion
soroban-sdk = { workspace = true, features = ["alloc"] }
```

Reply:
Ah, thanks that fixes the problem. Not sure why I only added it to the dev dependencies.